crates/turborepo-lib/src/task_graph/visitor.rs

```diff
@@ -172,6 +203,103 @@ impl<'a> Visitor<'a> {
 
         Ok(())
     }
+
+    fn sink(opts: &Opts) -> OutputSink<StdWriter> {
+        let err_writer = match matches!(
```

Comment:
Nit: could this be an `if` statement? `if matches!(...) && opts.run_opts.is_github_actions { ... } else { ... }` ?

Also, future note: we should probably always force everything to `stdout` on GH actions, even if we aren't grouping task logs. It's the closest we'll get to an in-order representation of what happened.